app/models/manageiq/providers/base_manager.rb

```diff
@@ -37,4 +37,8 @@ def self.options_description
       {}
     end
   end
+
+  def self.display_name(number = 1)
+    n_('Provider', 'Providers', number)
```

Comment:
Again this conflicts with the `Provider` model.  This is an `ExtManagementSystem`.